src/screens/welcome/Welcome.js

```diff
@@ -94,19 +113,28 @@ class Welcome extends Component {
     this.props.resetStory();
     this.props.resetMap();
     this.props.resetPhoto();
+    // Fetch the list of file paths
     await this.stateManager.getFilePaths();
+    // If the list of file paths is empty, set this.unfinishedReportsAbsent false
     this.unfinishedReportsAbsent = this.stateManager.filePaths.length == 0;
     if (this.state.link === "") {
```

Comment:
maybe add a comment above to indicate that this means the user did not provide a link to a new spreadsheet